docs/hello_nextflow/10_hello_seqera.md

```diff
@@ -1,7 +1,188 @@
+
+Seqera Platform uses the concept of **Compute Environments** to define the execution platform where a workflow will run.
+
+It supports the launching of workflows into a growing number of **cloud** and **on-premise** infrastructures.
+
+![Compute environments](img/compute_env_platforms.png)
+
+Each compute environment must be pre-configured to enable Seqera Platform to submit tasks. You can read more on how to set up each environment using the links below.
+
+!!! tip "The following guides describe how to configure each of these compute environments."
+
+    * [AWS Batch](https://docs.seqera.io/platform/24.1/compute-envs/aws-batch)
+    * [Azure Batch](https://docs.seqera.io/platform/24.1/compute-envs/azure-batch)
+    * [Google Batch](https://docs.seqera.io/platform/24.1/compute-envs/google-cloud-batch)
+    * [Google Life Sciences](https://docs.seqera.io/platform/24.1/compute-envs/google-cloud-lifesciences)
+    * [HPC (LSF, Slurm, Grid Engine, Altair PBS Pro)](https://docs.seqera.io/platform/24.1/compute-envs/hpc)
+    * [Amazon Kubernetes (EKS)](https://docs.seqera.io/platform/24.1/compute-envs/eks)
+    * [Google Kubernetes (GKE)](https://docs.seqera.io/platform/24.1/compute-envs/gke)
+    * [Hosted Kubernetes](https://docs.seqera.io/platform/24.1/compute-envs/k8s)
+
+### Selecting a default compute environment
+
+If you have more than one **Compute Environment**, you can select which one will be used by default when launching a pipeline.
+
+1. Navigate to your [compute environments](https://docs.seqera.io/platform/24.1/compute-envs/overview).
+2. Choose your default environment by selecting the **Make primary** button.
+
+**Congratulations!**
+
+You are now ready to launch workflows with your primary compute environment.
+
+### Launchpad
+
+Launchpad makes it easy for any workspace user to launch a pre-configured pipeline.
+
+![Launchpad](img/overview_launch.png)
+
+A pipeline is a repository containing a Nextflow workflow, a compute environment and workflow parameters.
+
+### Pipeline Parameters Form
+
+Launchpad automatically detects the presence of a `nextflow_schema.json` in the root of the repository and dynamically creates a form where users can easily update the parameters.
+
+!!! info
+
+    The parameter forms view will appear if the pipeline has a Nextflow schema file for the parameters. Please refer to the [Nextflow Schema guide](https://docs.seqera.io/platform/24.1/pipeline-schema/overview) to learn more about the schema file use-cases and how to create them.
+
+This makes it trivial for users without any expertise in Nextflow to enter their workflow parameters and launch.
+
+![Pipeline parameters](img/launch_rnaseq_nextflow_schema.png)
+
+### Adding a new pipeline
+
+Adding a pipeline to the pre-saved workspace launchpad is detailed in full on the [Seqera webpage docs](https://docs.seqera.io/platform/24.1/launch/launchpad#add-new-pipeline).
+
+In brief, these are the steps you need to follow to set up a pipeline.
```

Comment:
The below steps don't make complete sense to me- are you combining pipeline creation in Launchpad, and Launching? Needs a clear separation I think.